src/settings/components/NetworkPanel/Settings.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { func, string, object, arrayOf } from 'prop-types';
+import { settings } from '@cityofzion/neon-js';
+import { noop, map } from 'lodash';
+
+import Input from 'shared/components/Forms/Input';
+import Button from 'shared/components/Forms/Button';
+import Select from 'shared/components/Forms/Select';
+import Panel from 'shared/components/Panel';
+
+import Saved from './Saved';
+import { PREDEFINED_NETWORKS } from '../../values/networks';
+import styles from './Settings.scss';
+
+export default class Settings extends React.Component {
```

Comment:
We should call this NetworkPanel and change the filename to match.  Ideally the filename should match the parent folder name for each of these components.